<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceSettingsPage</class>
 <widget class="QWidget" name="AppearanceSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Appearance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="use_system_color_set">
        <property name="text">
         <string>Use the system default color set</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="use_a_custom_color_set">
        <property name="text">
         <string>Use a custom color set</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="select_foreground_color_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select foreground color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_foreground_color">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="select_background_color_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select background color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_background_color">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Background image</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="use_default_background">
        <property name="text">
         <string>Default background image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="use_album_cover_background">
        <property name="toolTip">
         <string>The album cover of the currently playing song</string>
        </property>
        <property name="text">
         <string>Album cover</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="use_no_background">
        <property name="text">
         <string>No background image</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="use_custom_background_image">
          <property name="text">
           <string>Custom image:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="background_image_filename">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_background_image_filename_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="select_background_blur_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Blur amount</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="blur_slider">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="background_blur_radius_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>0px</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="select_opacity_level_label">
          <property name="text">
           <string>Opacity</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="opacity_slider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="background_opacity_label">
          <property name="text">
           <string>40%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="moodbar_group">
     <property name="title">
      <string>Moodbars</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="moodbar_show">
        <property name="text">
         <string>Show a moodbar in the track progress bar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Moodbar style</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="moodbar_style"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="moodbar_save">
        <property name="text">
         <string>Save .mood files in your music library</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="moodbar_calculate">
        <property name="text">
         <string>Disable moodbar generation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Other</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="b_use_sys_icons">
          <property name="text">
           <string>Use system icons</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="b_hide_filter_toolbar">
          <property name="text">
           <string>Hide playlist filter toolbar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>use_a_custom_color_set</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_background_color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_a_custom_color_set</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_background_color_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_a_custom_color_set</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_foreground_color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_a_custom_color_set</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_foreground_color_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
